101-azure-machine-learning-service-workspace/README.md

```diff
@@ -0,0 +1,13 @@
+# Azure Machine Learning service workspace
```

Comment:
Use Title casing

Reply:
Sadly, that is title casing for this service. The official service name is "Azure Machine Learning service", with a lower case 's' in 'service'. https://azure.microsoft.com/en-us/services/machine-learning-service/ is a marketing page where you can see the same capitalization.

The workspace is the central artifact for working with the service.